rcl/test/rcl/test_graph.cpp

```diff
@@ -1321,3 +1330,172 @@ TEST_F(CLASSNAME(TestGraphFixture, RMW_IMPLEMENTATION), test_rcl_service_server_
   wait_for_service_state_to_change(false, is_available);
   ASSERT_FALSE(is_available);
 }
+
+
+/*
+ * This does not test content of the response.
+ * It only tests if the return code is the one expected.
+ */
+TEST_F(CLASSNAME(TestGraphFixture, RMW_IMPLEMENTATION),
+  test_rcl_get_publishers_info_by_topic_null_node)
+{
+  rcl_allocator_t allocator = rcl_get_default_allocator();
+  auto ret = rcl_get_publishers_info_by_topic(nullptr,
+      &allocator, this->topic_name, false, this->topic_info_array);
+  EXPECT_EQ(RCL_RET_NODE_INVALID, ret);
+}
+
+/*
+ * This does not test content of the response.
+ * It only tests if the return code is the one expected.
+ */
+TEST_F(CLASSNAME(TestGraphFixture, RMW_IMPLEMENTATION),
+  test_rcl_get_subscriptions_info_by_topic_null_node)
+{
+  rcl_allocator_t allocator = rcl_get_default_allocator();
```

Comment:
```c++
auto allocator = rcl_get_default_allocator();
const auto ret = ...
```

same as above; `const` is not really needed on `ret`, however it indicates to the reviewer that it is not reassigned later inside the function.
`allocator` cannot be `const` unless it and its fields can be accessed as `const`.